liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/TestStartServerMojo.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * (C) Copyright IBM Corporation 2014, 2017.
```

Comment:
I removed TestStopServerMojo.java because it didn't really provide any functionality